backend/parse.js

```diff
@@ -31,11 +31,28 @@ export function parseSQL(data) {
 
     // iterate over each object (each object becomes a row)
     // returns: a row of rows (table)
-    const rows = data.map(row =>
-        Object.keys(row).map(
-            columnname => row[columnname]
-        )
-    );
+    const rows = [];
+    for (let i = 0; i < data.length; i++) {
+        rows[i] = [];
+        for (let j = 0; j < columnnames.length; j++) {
+            let cell = data[i][columnnames[j]];
+            // Like PostGIS's GeoJSON or Dates
+            const cellType = type(cell);
+            if (contains(cellType, ['Object', 'Boolean'])) {
```

Comment:
OK finally connected. I think the `aapl_2014` table is the only one with dates. Looks OK:
![image](https://cloud.githubusercontent.com/assets/1280389/21089410/d0d7f2ca-c005-11e6-8460-2055be2ea5d4.png)